src/assets/img/icons/plusSign.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M8 6V1.00685C8 0.449949 7.55228 0 7 0C6.44386 0 6 0.45078 6 1.00685V6H1.00685C0.449949 6 0 6.44772 0 7C0 7.55614 0.45078 8 1.00685 8H6V12.9932C6 13.5501 6.44772 14 7 14C7.55614 14 8 13.5492 8 12.9932V8H12.9932C13.5501 8 14 7.55228 14 7C14 6.44386 13.5492 6 12.9932 6H8Z" fill="white"/>
+</svg>
```

Comment:
We already have an icon `add` which is that plus sign

Reply:
And if you don't mind maybe name of icon should be `plus_sign.svg` like others